projects/WayneStateU_TDA.yaml

```
@@ -0,0 +1,13 @@
+Name: WayneStateU_TDA
+Description: Topological Data Analysis of fMRI Signals during Learning: Function to Structure
```

```suggestion
Description: "Topological Data Analysis of fMRI Signals during Learning: Function to Structure"
```
Looks like this needs to be quoted because it has a colon in it.